client/app/scripts/superdesk/superdesk.js

```diff
@@ -97,8 +97,13 @@ define([
         }]);
 
     // make sure there is templates module defined
-    angular.module('templates', []);
-    modules.push('templates');
+    try {
```

Comment:
Yes. Or we could copy the same try-catch in templates-cache to remove this constraint if it's important